.gitignore

```
@@ -0,0 +1 @@
+result
```

Might as well also ignore created disk images
```suggestion
result
9front.*.qcow2
```